packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -16,10 +16,39 @@ class _BaseConfig:
     keep_untouched = (cached_property,)
 
 
+class ContainerSpec(BaseModel):
+    """Implements entries can can be overriden for https://docs.docker.com/engine/api/v1.41/#operation/ServiceCreate
+    request body: TaskTemplate -> ContainerSpec
+    """
+
+    command: list[str] = Field(
+        alias="Command",
+        description="The command to be run in the image. If None, it will not override default 'run'",
+        # NOTE: currently constraint to our use cases. Might mitigate some security issues.
+        min_items=1,
+        max_items=2,
+    )
+
+    class Config(_BaseConfig):
+        schema_extra = {
+            "examples": [
+                {"Command": ["executable"]},
+                {"Command": ["executable", "subcommand"]},
+                {"Command": ["ofs", "linear-regression"]},
+            ]
+        }
+
+
 class SimcoreServiceSettingLabelEntry(BaseModel):
+    """These value is used to build the request body of https://docs.docker.com/engine/api/v1.41/#operation/ServiceCreate
```

Comment:
```suggestion
    """These value are used to build the request body of https://docs.docker.com/engine/api/v1.41/#operation/ServiceCreate
```

Reply:
well, if one day we move to kubernetes, this comment might not be really necessary here. but that is ok.